### ISSUE TITLE
Distance calculator doesn't work with accessibility plugin

### ISSUE THREAD

**Issue by vshcherb** [osmandapp/Osmand#3830]
Distance calculator doesn't work with accessibility plugin. 
We need to find workaround for that and better fix (lots of requests through support).

**Comment by vshcherb** [comment]
Not reproducible
